core/Cargo.toml

```diff
@@ -19,14 +19,15 @@ links = "nix"
 bench = false
 
 [features]
-default = ["markdown", "repl", "doc", "format", "spanned-deser"]
+default = ["markdown", "repl", "doc", "format", "spanned-deser", "bytecode-experimental"]
```

Comment:
TODO: temporary change to make rust-analyzer work by default. Change that before merging.